sphinx-doc/features.rst

```diff
@@ -0,0 +1,139 @@
+constraints (`md.methods.rattle`). HOOMD-blue provides a number of pair potentials (`md.pair`)
+including pair potentials that depend on particle orientation (`md.pair.aniso`) and many body
+potentials (`md.many_body`). HOOMD-blue also provides bond potentials and distance constraints
+commonly used in atomistic and coarse-grained force fields (`md.angle`, `md.bond`,
+`md.constrain.Distance`, `md.dihedral`, `md.improper`, `md.special_pair`) and can model rigid bodies
+(`md.constrain.Rigid`). External fields `md.external.field` apply potentials based only on the
+particle's position and orientation, including walls (`md.external.wall`) to confine particles in a
+specific region of space. `md.long_range` provides long ranged interactions, including the PPPM
+method for electrostatics. HOOMD-blue enables active matter simulations with `md.force.Active` and
+`md.update.ActiveRotationalDiffusion`. At runtime, `hoomd.version.md_built` indicates whether the
+build supports MD simulations.
+
+md.update
+
+.. seealso::
+
+    Tutorial: :doc:`tutorial/01-Introducing-Molecular-Dynamics/00-index`
+
+Python package
+--------------
+
+HOOMD-blue is a Python package and is designed to interoperate with other packages in the scientific
+Python ecosystem and to be extendable in user scripts. To enable interoperability, all operations
+provide access to useful computed quantities as properties in native Python types or numpy arrays
+where appropriate. Additionally, `State <hoomd.State>` and `md.force.Force` provide direct access to
+particle properties and forces using Python array protocols. Users can customize their simulation or
+extend HOOMD-blue with functionality implemented in Python code by subclassing `trigger.Trigger`,
+`variant.Variant`, `hoomd.update.CustomUpdater`, `hoomd.write.CustomWriter`,
+`hoomd.tune.CustomTuner`, or by using the HOOMD-blue API in combination with other Python packages
+to implement methods that couple simulation, analysis, and multiple simulations (such as umbrella
+sampling).
+
+.. seealso::
+
+    Tutorial: :doc:`tutorial/04-Custom-Actions-In-Python/00-index`
+
+CPU and GPU devices
+-------------------
+
+HOOMD-blue can execute simulations on CPUs or GPUs. Typical simulations run more efficiently on
+GPUs for system sizes larger than a few thousand particles, although this strongly depends on the
+details of the simulation. The provided binaries support NVIDIA GPUs. Build from source to enable
+preliminary support for AMD GPUs. CPU support is always enabled. GPU support must be enabled at
+compile time with the ``ENABLE_GPU`` CMake option (see :doc:`building`). Select the device to use at
+run time with the `device <hoomd.device>` module. Unless otherwise stated in the documentation,
+**all** operations and methods support GPU execution. At runtime, `hoomd.version.gpu_enabled` indicates
+whether the build supports run time compilation.
```

Comment:
Does run time compilation mean JIT code?

Or does it mean "able to run on GPU"?

```suggestion
whether the build supports running on the GPU.
```

Reply:
Fixed.